controls/nginx_service.rb

```diff
@@ -0,0 +1,23 @@
+title 'Nginx base plan tests'
+
+plan_name = input('plan_name', value: 'nginx')
+plan_ident = "#{ENV['HAB_ORIGIN']}/#{plan_name}"
+
+control 'base-plans-nginx-service' do
+    impact 1.0
+    title 'Ensure Nginx is running as a service'
+    desc ''
+
+    # Get the ident of the package
+    status_cmd = command("hab svc status #{plan_ident}")
+    svc_status = status_cmd.stdout.strip.split(/\s+/)
+
+    # Test that the service is running
+    # Ideally this should test the type, desired and state - however
```

Comment:
Entirely up to you @russellseymour, but if you look in the memcached or minio plans you will find a regex that will check whether the service is up:

``its('stdout') { should match /(?<package>#{plan_pkg_ident})\s+(?<type>standalone)\s+(?<desired>up)\s+(?<state>up)/ }``

Reply:
Thanks have added this :+1: